news/2022-09-27-changes-to-osu-taiko-sr-and-pp.md

```diff
@@ -0,0 +1,153 @@
+
+Previous approaches aimed to scale maps in regards to the [difficulty naming hierarchy](/wiki/Ranking_Criteria/Difficulty_naming), while this rework has deviated away from that goal, aiming instead to reflect true difficulty regardless of naming practices. 
+
+This also means that star rating values for maps with near-FCs are no longer near 9.5 stars, but rather can expand well into the 10 star range (medal hunters rejoice!).
+
+## Changes to performance points
+
+In recent years, osu!taiko has lagged behind the other three game modes in terms of raw performance point values, to the tune of being roughly 40% lower overall. In addition, the survey indicated a great preference for re-evaluating the way performance points are calculated.
+
+Previously, performance points in osu!taiko were calculated by summing the "accuracy pp" and "difficulty pp" components of a score together. The following changes should serve to increase these values, make them more relevant to the apparent difficulty of a given map, and improve balance not only between difficulty-increasing mods, but also their difficulty-reducing counterparts.
+
+All of these changes aim to continue the goals outlined in the new star rating rework.
+
+### Globally applied changes
+
+The final step of performance points calculation involves applying a simple multiplier to the calculated value, which consists of both difficulty and accuracy performance points. 
+
+This allows for simple balancing changes, and works alongside specific multipliers found within their respective separate difficulty values. To begin with, we have made the following changes:
+
+- The Hidden multiplier has been decreased from **1.10x** to **1.075x**, as a further bonus was placed in difficulty pp, rather than globally.
+- Easy now applies a **0.975x** multiplier to help address its problematic nature, especially when paired with DoubleTime.
+- The NoFail multiplier has been removed. osu!taiko uses a different HP system compared to osu!, negating the need for any kind of penalty. This also helps prevent "sandbagging" in lower ranks.
+
+Additionally, the community survey highlighted issues with short maps being either underweighted at a full combo, or overweighted on plays with several misses.
+
+To combat this, misses will now significantly affect difficulty pp on maps with a max combo below 1000, scaling the miss count by a multiplier of `1000 / total successful hits`. This may significantly impact plays on shorter maps with a higher miss count, as visualized below.
+
+![Effective miss count graph](/wiki/shared/news/2022-09-27-changes-to-osu-taiko-star-rating-and-performance-points/effective-miss-count.jpg)
+
+### Changes to difficulty performance points values
+
+The following changes should result in impressive scores on harder beatmaps rewarding more appropriate performance point values, with top player [syaron105](https://osu.ppy.sh/users/8741695) gaining over 6400 total pp!
```

Comment:
I'm confused. All the points below are small changes (mostly nerf) so how did a user gain 6400 pp as a result? Something is missing in the explanation.

Reply:
i'll rejiggle this around to make more sense